### PR TITLE
Bump bdk version to 1.0.0-alpha.10

### PR DESCRIPTION
**Pull request by danielabrozzoni**
### Description

bdk_chain to 0.13.0
bdk_bitcoind_rpc to 0.9.0
bdk_electrum to 0.12.0
bdk_esplora to 0.12.0
bdk_file_store to 0.10.0
bdk_testenv to 0.3.0
bdk_persist to 0.2.0

#### All Submissions:
- [x] I've signed all my commits
- [x] I followed the [contribution guidelines](https://github.com/bitcoindevkit/bdk/blob/master/CONTRIBUTING.md)
- [x] I ran cargo fmt and cargo clippy before committing

### CODE REVIEW

File: crates/file_store/Cargo.toml [hunk 1]
```diff
@@ -12,8 +12,8 @@ readme = "README.md"
 
 [dependencies]
 anyhow = { version = "1", default-features = false }
-bdk_chain = { path = "../chain", version = "0.12.0", features = [ "serde", "miniscript" ] }
-bdk_persist = { path = "../persist", version = "0.1.0"}
+bdk_chain = { path = "../chain", version = "0.13.0", features = [ "serde", "miniscript" ] }
+bdk_persist = { path = "../persist", version = "0.2.0"}
```
Comment (notmandatory) [inline crates/file_store/Cargo.toml]:
```suggestion
bdk_persist = { path = "../persist", version = "0.1.0"}
```

File: crates/persist/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "bdk_persist"
 homepage = "https://bitcoindevkit.org"
-version = "0.1.0"
+version = "0.2.0"
```
Comment (notmandatory) [inline crates/persist/Cargo.toml]: 🔧 Don't need to bump this one since it's the first release. 
```suggestion
version = "0.1.0"
```

**Comment by notmandatory** [comment]
Everything looks good, just one small change and can be merged.

**Review by notmandatory — APPROVED** [review]
ACK d2e0f64d1c4bce84170e929c46643e3d167db948

Needs those couple changes but ACKing now so you can merge when you're ready.

**Review by storopoli — APPROVED** [review]
ACK 2cda9f44ee1807898e4f2afffc95774839fee678

**Pull request merged**